book/modules/mod-A/pages/shared

```diff
@@ -0,0 +1 @@
+../../shared/pages/shared
```

Comment:
Just a symlink to the actual content in `modules/shared/`.  Just to make antora think that these files are really inside the consuming module (`mod-A`).